### PR TITLE
Allows Species to set custom say spans.

### PR DESCRIPTION
**Pull request by RemieRichards**
- Species may now set custom say spans
- Skeletons will now speak in either Sans or Papyrus
  SansAndPapyrus

### CODE REVIEW

**Comment by Xhuis** [comment]
If they speak in Papyrus, it should be all caps. In Sans, it should be all lowercase.

**Comment by RemieRichards** [comment]
@Xhuis I think that's taking it a little far, plus it's technically impossible atm since the first letter is auto uppercased deep inside Say code.

**Comment by Steelpoint** [comment]
Anyone within seven tiles of a Skeleton named Papyrus should start automaticly playing this theme music.

**Comment by Anonus** [comment]
`NYEH HEH HEH`

**Comment by Koriath** [comment]
How exactly did you guys make Sans shorter? Can you set height in /tg/?

**Comment by RemieRichards** [comment]
@Koriath the resize var, it was added by AnturK a while back, you guys *should* have it, but it's a simple port, I just set his Resize to 0.8, same as the dwarf mutation.

**Comment by Koriath** [comment]
Thankashen. Will come in handy to make fly people smallish.

**Comment by Ricotez** [comment]
papyrusblush

i love you for this

**Comment by Jacquerel** [comment]
This is funny and all, and I love Undertale, but there's no way this is going to be a lasting feature. It will get tired after literally two rounds.  
You don't really want skeletons to speak Papyrus and Comic Sans for all time. The reference is not worth it.

**Comment by RemieRichards** [comment]
@Jacquerel code is not set in obsidian stone, it can be changed if it becomes unwanted down the line.

**Comment by Jacquerel** [comment]
I know it can, but that doesn't mean making changes that you *know* are going to be removed again down the line is a good idea.

**Comment by RemieRichards** [comment]
Also @Jacquerel think about how common skeletons are.
Not very common, 1 necrostone per summon magic, and wizards can only buy 5 stones. that's a max of 5 (cant take 5 + summon) Not to mention they tend not to buy the stones (idk why it's free minions with 40 brute swords)

tl;dr it's fine.

**Comment by PKPenguin321** [comment]
:+1:

**Comment by Ricotez** [comment]
the undertale reference is just one part of this commit, the Big New Feature is that you can now set a span per species, so if you want to make lizards even more obnoxious you could make all their text big bold red

**Comment by Steelpoint** [comment]
Would it be possible to set up a system to allow this to change the text of someone speaking based on any mask slot items they are wearing?

I'm more imagining the Death Squad get a more menacing appearing text speaking, to simulate to a extent a unfriendly communication from their suits. Or even for gimmicks admins want to do.

**Comment by Ricotez** [comment]
easily so, give masks a mask_get_spans() proc or something, check in /mob/living/carbon/human/get_spans() if the human is wearing a mask, and if so call that proc too

**Comment by Xhuis** [comment]
\>not suggestions get_sans()

**Comment by RogerWilcoBeep** [comment]
CRACK THE FUCKING ACK.

**Comment by PKPenguin321** [comment]
Necropost meme
